data/sql_setup/prepare_schedule_d.sql

```diff
@@ -0,0 +1,19 @@
+-- Create trigger to maintain Schedule C search columns
+ALTER TABLE disclosure.nml_sched_d DROP COLUMN if EXISTS creditor_debtor_name_text;
+
+ALTER TABLE disclosure.nml_sched_d add column creditor_debtor_name_text tsvector;
```

Comment:
Ah okay, I see what you mean with the `UPDATE` statement below - I'm not 100% sure myself, was just basing that off of the [PostgreSQL documentation on altering a table](https://www.postgresql.org/docs/9.5/static/ddl-alter.html).